src/components/GoodList/GoodList.jsx

```diff
@@ -0,0 +1,10 @@
+
+export const GoodList = ({ goods }) => (
+  goods.map(good => (
+    <ul>
+      <li data-cy="Good" key={good}>
+        {good}
```

Comment:
```suggestion
<ul>
  goods.map(good => (
      <li data-cy="Good" key={good}>
        {good}
```

Reply:
<img width="725" alt="Screenshot 2024-01-14 at 15 58 52" src="https://github.com/mate-academy/react_list-of-goods-js/assets/135232001/b8361f1b-dc3d-4bc7-a79c-c11f8310c2ba">
Consider fixing the layout

Reply:
 Please fix [this comment](https://github.com/mate-academy/react_list-of-goods-js/pull/589/files#r1451738935)
 ul should be outside of map

Reply:
if I remove data-cy="Good"
i have error test (((
i dont understand why?
![Screenshot_6](https://github.com/mate-academy/react_list-of-goods-js/assets/100188879/54a83b79-528e-4c3e-a111-74f42abc5c6e)